### PR TITLE
feat : completed alert component

### PR DESCRIPTION
**Pull request by shaikhhamza11**
Description:
1) Added Alert component
Screenshot (16)

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  border-left: 0.8rem solid #bababa;
+  background: #ededed;
+  color: #a1a1a1;
+  border-radius: 0.5rem;
+  display: flex;
+  align-items: center;
+  text-align: left;
+  font-size: 1.6rem;
+  align-content: center;
+  flex-wrap: nowrap;
```
Comment (junaidshaikh-js) [inline components/alert.css]: flex-wrap has default  `nowrap` value

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  margin-right: 1rem;
+}
+.alert .alert-content {
+  width: 100%;
+  font-weight: bolder;
+}
+.alert .alert-close {
+  margin-left: 1rem;
+}
+
```
Comment (junaidshaikh-js) [inline components/alert.css:31]: Use consistent formatting in your file.

File: components/alert.css (hunk 1 above)
Comment (shaikhhamza11) [reply components/alert.css]: Thanks. I have removed that

File: components/alert.css (hunk 2 above)
Comment (shaikhhamza11) [reply components/alert.css:31]: I have prettier installed and code is formatted . Can you explain this please.

**Review by KanekarSnehal — APPROVED** [review]
Suggestion: use css variables for colors.
rest all looks good you can merge.

**Pull request merged**